src/pages/admin/rental-management/RentalManagement.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import MenuBtn from '../../../components/Btn/MenuBtn';
+import styled from 'styled-components';
+import Header from '../../../components/Header/Header';
+
+const Wrapper = styled.div`
+  height: 100vh;
+`;
+const ContentWrapper = styled.div`
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  padding: 50px 75px;
+  width: 100%;
+  height: 100%;
+`;
+const Title = styled.h3`
+  color: #000;
+  font-family: Pretendard;
+  font-size: 25px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: normal;
+  margin: 0;
+`;
+const BtnWrapper = styled.div`
+  display: flex;
+  height: 100%;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 19px;
+`;
+
+const RentalManagement = () => {
+  return (
+    <Wrapper>
+      <Header />
+      <ContentWrapper>
+        <Title>신청 내역 관리</Title>
+        <BtnWrapper>
+          <MenuBtn type="button" children="물품 별로 보기" />
```

Comment:
```suggestion
          <MenuBtn type="button">물품 별로 보기</MenuBtn>
```

아마 이렇게도 동작할 것 같은데, js 리액트는 처음이라서 잘 모르겠네요..